routes/tasks.go

```diff
@@ -0,0 +1,111 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/labstack/echo"
+	"net/http"
+
+	m "github.com/kirikami/go_exercise_api/database/models"
+	u "github.com/kirikami/go_exercise_api/utils"
+)
+
+func (h ApiV1Handler) SaveTaskHandler(c echo.Context) error {
+	task := m.Task{}
+
+	if err := c.Bind(&task); err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError)
+	}
+
+	if err := h.DB.Save(&task).Error; err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError)
+	}
+
+	return c.JSON(http.StatusCreated, task)
+}
+
+func (h ApiV1Handler) UpdateTaskHandler(c echo.Context) error {
+	idParam := c.Param("id")
+
+	id, err := u.ParseIdInt64FromString(idParam)
+
+	if err != nil {
+		return echo.NewHTTPError(http.StatusBadRequest)
+	}
+
+	task := m.Task{}
+
+	if err := h.DB.First(&task, id).Error; err != nil {
+		return echo.NewHTTPError(http.StatusInternalServerError)
+	}
+
+	if err := c.Bind(&task); err != nil {
+		c.Logger().Error(err)
```

Comment:
Please check this https://github.com/labstack/echo/blob/master/echo.go#L230 You can write your own error handler and return from handler custorm error.
For example `return helpers.SendInternalError(err, "User message")`. Inside custom error handler you can make type assertion, then log error for developer and return some general message for a client. `return c.JSON(http.StatusInternalServerError , ErrorType`)